server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/WithCassandraBlobStore.java

```diff
@@ -33,7 +40,18 @@
     private final JamesServerExtension jamesServerExtension;
 
     WithCassandraBlobStore() {
-        jamesServerExtension = CassandraRabbitMQJamesServerFixture.baseExtensionBuilder().build();
+        jamesServerExtension = new JamesServerBuilder()
+            .extension(new DockerElasticSearchExtension())
+            .extension(new CassandraExtension())
+            .extension(new RabbitMQExtension())
+            .server(configuration -> GuiceJamesServer
+                .forConfiguration(configuration)
+                .combineWith(CassandraRabbitMQJamesServerMain.baseModule(cassandra()))
+                .overrideWith(TestJMAPServerModule.limitToTenMessages())
+                .overrideWith(JmapJamesServerContract.DOMAIN_LIST_CONFIGURATION_MODULE)
+                .overrideWith(new BlobStoreCacheConfiguredModulesSupplier.CacheDisabledModule())
```

Comment:
with it's needed at all, I  think we said that tests should rely on the fact the server now how to instantiate the right set of modules based on the configuration.

Reply:
It's not needed

Reply:
It is a mistake, I restricted the visibility of such modules to ensure that we never use it and relies on the product instead.